File: Assignment_4/Assignment4_2.py
"""-------------------------------------------------------------------------------------------
                          Assignment4_2
                    (Student name - Vaishali Jorwekar)
                     Python by Marvellous Infosystems
---------------------------------------------------------------------------------------------
Problem statement: This program contains a lambda function and returns multiplication
------------------------------------------------------------------------------------------"""

Multi = lambda num1,num2 : num1 * num2
#------------------------------------------------------------------------------------------
# This function accepts a input number from user and calls lambda multiplication function
#------------------------------------------------------------------------------------------
def main():

    number1=int(input("Enter first number :"))
    number2=int(input("Enter second number :"))

    answer = Multi(number1,number2)
    print("--------------------------------------------------------------")
    print(f"Multiplication of {number1} * {number2} : {answer}")
    print("--------------------------------------------------------------")

#-----------------------------------------------------------------------------------
# This is main entry point of the program
#-----------------------------------------------------------------------------------
if __name__=="__main__":
    main()
#-----------------------------------------------------------------------------------

File: JavaPractice/Assignment_4/Assignment4_2.java
import java.util.Scanner;

/*-------------------------------------------------------------------------------------------
                          Assignment4_2
                    (Student name - Vaishali Jorwekar)
                     Python by Marvellous Infosystems
---------------------------------------------------------------------------------------------
Problem statement: This program contains a lambda function and returns number power of 2
------------------------------------------------------------------------------------------*/

public class Assignment4_2{

	
	public static void main(String[] args) {
		Scanner inputObj = new Scanner(System.in);
		System.out.print("Enter First Number:"); 
		int number = inputObj.nextInt();
		
		MathsFunction square=  (num) ->num*num;
		int result=square.Square(number);
		System.out.println("Square of "+number+" is :"+result);
		
		
		inputObj.close();
	}

	
}
